### PR TITLE
Update to 1.4.8

### PR DESCRIPTION
**Pull request by gillins**
New version supports out of tree builds and uses `FindHDF5.cmake` etc.

### CODE REVIEW

**Comment by gillins** [comment]
@conda-forge-admin, please re-render

**Comment by ocefpaf** [comment]
B/c we are using `python` to activate the `vc` on Windows the re-rendering with latest conda-smithy is not OK. It created a Python build matrix on Linux and OS X as well. We need to convert this recipe to the new format.

**Comment by gillins** [comment]
@ocefpaf does this look ok?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,39 +1,34 @@
...
 build:
-    number: 4
-    skip: True  # [win and py36]
-    features:
-        - vc9  # [win and py27]
-        - vc14  # [win and py35]
+    number: 0
+    run_exports:
+      # should match >=1.4.7,<1.5 ?
+      - {{ pin_subpackage('kealib', min_pin='1.4.7', max_pin='x.x') }}
```
Comment (ocefpaf) [inline recipe/meta.yaml]: @gillins I guess we could use this opportunity to sync with the Anaconda recipe that already uses conda-build 3 format: https://github.com/AnacondaRecipes/kealib-feedstock/blob/master/recipe/meta.yaml

Ping @msarahan

**Comment by ocefpaf** [comment]
LGTM! Any additional changes need to wait for `hdf5` to be in the same format.

@msarahan do you mind taking a quick look?

**Comment by msarahan** [comment]
hdf5 should be:

```
requirements:
  host:
    - hdf5
  run:
    - hdf5
```

Ideally you would not need the run entry.  Because we don't have run_exports on the conda-forge hdf5 packages, you do need to list it.  You do not need to list the version, though, because it'll be pinned according to https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L279 

It will be constrained to 3 places by virtue of https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L106

It will end up rendered as something like this:

```
requirements:
  host:
    - hdf5 1.10.2 h134373_2
  run:
    - hdf5 >=1.10.2,<1.10.3.0a0
```

**Comment by msarahan** [comment]
PS sorry for delay, I missed this earlier.

**Pull request merged**